test/features/major_test.js

```diff
@@ -0,0 +1,59 @@
+
+    withUsersInTheDatabase(function(){
+      it('runs through scenario of a player creating a prrr and a coach seeing a new prrr being added and claims it', function(){
+        this.timeout(60*10000)
+
+        const player = this.createBrowser()
+        const coach = this.createBrowser()
+
+        sinon.stub(Queries.prototype, "getPullRequest").returns(Promise.resolve({FAKE_PR: true}));
+
+        return Promise.resolve()
+          //player logs in
+          .then(_ => player.loginAs(15825329))
+          //player goes to homepage
+          .then(_ => player.visit('/'))
+          //coach logs in
+          .then(_ => coach.loginAs(47313))
+          //coach goes to homepage
+          .then(_ => coach.visit('/'))
+          //player clicks add Prrr button, then redirects to addPrrr page
+          .then(_ => player.wait(until.elementLocated(By.className('Navbar-button-add-prrr')), 2000).click())
+          //player pastes pull request url into text box
+          .then(_ => player.findElement(By.className('RequestReviewPage-InputBox'))
+            .sendKeys('https://github.com/AbrahamFergie/Obeisant-Gecko/pull/6'))
+          //player once seeing fields are populated appropriately, then clicks add
+          .then(_ => player.findElement(By.className('RequestReviewPage-Button')).click())
+          //player notices that current pull request is under my requested prrrs with pull request address and request by information and opens links
+          .then(_ => player.wait(until.elementLocated(By.xpath("//td[@class='PrrrsTable-pr']//a[@href='https://github.com/AbrahamFergie/Obeisant-Gecko/pull/6']")), 2000).click())
+          .then(_ => player.wait(until.elementLocated(By.xpath("//td[@class='PrrrsTable-requested']//a[@href='https://github.com/AbrahamFergie']")), 2000).click())
+          //closes popup tabs
+          .then(_ => player.closeTabs())
+          //coach upon seeing that a Prrr has been placed in queue, then clicks button to claim a Prrr
+          .then(_ => coach.wait(until.elementLocated(By.className('Button-default ReviewButton')), 2000).click())
+          //finds tabs as they open
+          .then(_ => coach.wait(coach.identifyTabPopup(), 2000))
+          //closes said tabs
+          .then(_ => coach.wait(coach.closeTabs(), 2000))
+          //coach completes Prrr review upon a substantial inquiry into the pull request adding comments and smiley faces in appropriate places
+          .then(_ => coach.wait(until.elementLocated(By.xpath(
+              "//*[@class='ClaimAPrrr-UserClaimedAPrrr-Buttons']//button[contains(.,'Complete')]")
+            ),2000).click())
+          //player sees that prrr has been completed
+          .then(_ => player.wait(until.elementLocated(By.xpath("//td//span[contains(.,'a few seconds ago')]"))))
+          //coach toggles My Reviewed table
+          .then(_ => coach.findElement(By.xpath("//table[@class='PrrrsTable MyReviewedPrrrs']//td//span[contains(.,'a few seconds ago')]")))
```

Comment:
you are clicking the button but you need to assert something for it to be a test. 

```
the page should have this text "xxxxx"
click this button
the page should not have this text "xxxxx"
```